### PR TITLE
Fix support for PROGRAM-DATE-TIME tag and other fun things

### PR DESCRIPTION
**Pull request by mjneil**
Support for adding PROGRAM-DATE-TIME tags was not functioning properly, so this fixes it. Note that this only fixes it for vod playlists where every segment matches the target duration of the playlist. If there are segments that have a different duration, overtime the program-date-time values will shift. Fixing to use durations would be rather simple (just sum durations of segments from `i` to `position`) but I wanted to keep this pr short

I also added 2 other unrelated features
1. If the VOD playlist already contains a DISCONTINUITY-SEQUENCE tag, murphy will use the value of that tag as the starting discontinuity sequence value rather than 0.
1. I added 2 buttons to the ui page, "trigger 404 for every playlist response" and "trigger 200 for every playlist response" for better control of playlist errors

### CODE REVIEW

File: live-hls.js [hunk 1]
```diff
@@ -345,6 +344,12 @@ const extractResourceWindow = function(mfest, duration, event, streamtype) {
   debuglog('event start ' + event.start);
 
   event.discomod = 0;
+
+  if (header.Discontinuity.value) {
```
Comment (tomruggs) [inline live-hls.js]: Is this safe?  Should it be `(header.Discontinuity && header.Discontinuity.value)`?

Comment (mjneil) [reply live-hls.js]: It should be, the properties are created when the header object is created 
https://github.com/videojs/murphy/blob/master/live-hls.js#L74

Comment (tomruggs) [reply live-hls.js]: Then `event.discomod = parseInt(header.Discontinuity.value, 10) || 0;`?

**Comment by mrocajr** [comment]
LGTM after 346 is deleted.

**Review by tomruggs — APPROVED** [review]

**Pull request merged**